<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="MDC-Alexeid-M" failures="1" tests="5" name="Surefire test" time="0.8" errors="0" timestamp="2024-06-17T10:46:13 EEST">
  <testcase classname="student.examples.PasswordProcessTest" name="testPasswordStrength" time="0.006"/>
  <testcase classname="student.examples.PasswordProcessTest" name="testPasswordStrength" time="0.001"/>
  <testcase classname="student.examples.PasswordProcessTest" name="testPasswordStrength" time="0.001"/>
  <testcase classname="student.examples.PasswordProcessTest" name="testPasswordStrength" time="0.001"/>
  <testcase classname="student.examples.PasswordProcessTest" name="testPasswordStrength" time="0.006">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at student.examples.PasswordProcessTest.testPasswordStrength(PasswordProcessTest.java:54)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 34 stack frames]]>
    </failure>
  </testcase> <!-- testPasswordStrength -->
</testsuite> <!-- Surefire test -->
